custom_components/bhyve/switch.py

```diff
@@ -515,6 +525,27 @@ def is_on(self):
         """Return the status of the sensor."""
         return self._is_on
 
+    async def set_smart_watering_soil_moisture(self, percentage):
+        """Set the soil moisture percentage for the zone."""
+        if self._attrs[ATTR_SMART_WATERING_ENABLED]:
+            landscape = await self._bhyve.get_landscape(self._device_id, self._zone_id)
```

Comment:
The BHyve API is notoriously flakey (it's probably not designed with home assistant's long lived websockets in mind...) and as such often 500s.

Can you please wrap this API call (and the one below) in a `try / except` and handle this gracefully by logging a warning.

Reply:
Will do.